player/player.gd

```diff
@@ -129,6 +129,7 @@ func die():
 		return
 	dead = true
 
+	Input.set_mouse_mode(Input.MOUSE_MODE_VISIBLE)
```

Comment:
It would be clearer to do this in `global.set_game_over()`.

Reply:
Actually no, all mouse mode stuff is handled in `level/ui.gd` currently, so best move it there (so in `game_over()`).